units/src/amount/unsigned.rs

```diff
@@ -70,12 +70,26 @@ impl Amount {
     pub const SIZE: usize = 8; // Serialized length of a u64.
 
     /// Constructs a new [`Amount`] with satoshi precision and the given number of satoshis.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use bitcoin_units::amount::Amount;
```

Comment:
I rekon we should use `use bitcoin_units::Amount` to highlight that is the easiest way to import this type.

Reply:
Done